src/Database/LSMTree/Internal/Index/Ordinary.hs

```diff
@@ -4,13 +4,7 @@
 module Database.LSMTree.Internal.Index.Ordinary
 (
     IndexOrdinary (IndexOrdinary),
-    toLastKeys,
-
-    -- * Search
-    search,
-
-    -- * Deserialisation
-    fromSBS
```

Comment:
Same comment about keeping the exports

Reply:
Please see [my below general comment on specialization](https://github.com/IntersectMBO/lsm-tree/pull/476#issuecomment-2491953851).